src/figuresClasses.ts

```diff
@@ -1,11 +1,77 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
- shape is not a string, available values "triangle, circle or rectangle"
- color too, "red, green or blue"
![image](https://github.com/mate-academy/ts_figures_classes/assets/63243136/a8e96bb1-1d61-4dcf-94da-739dce98dfb8)